presentation/src/main/java/org/cryptomator/presentation/util/BiometricAuthentication.kt

```diff
@@ -45,7 +45,7 @@ class BiometricAuthentication(val callback: Callback, val context: Context, val
 		val biometricAuthCryptor: BiometricAuthCryptor
 
 		try {
-			biometricAuthCryptor = BiometricAuthCryptor.getInstance(context)
+			biometricAuthCryptor = BiometricAuthCryptor.getInstance(context, org.cryptomator.util.crypto.CryptoMode.GCM)
```

Comment:
Because of https://github.com/cryptomator/android/blob/develop/presentation/src/main/java/org/cryptomator/presentation/util/BiometricAuthentication.kt#L29-L31